### DISCUSSION TITLE
fastq-dump -X 1 -Z SRR1553591 Error

### DISCUSSION THREAD

**Discussion by Kessler111** [biostars/biostar-handbook-issues#222] [Q&A] [answered]
EDIT: I fixed the issue. Just added the export PATH command in bash_profile. But for some reason every new terminal I open I have to run source~/.bash_profile, otherwise it still doesn't work. I made by bio terminal startup by running source ~/.bash_profile and then conda activate bioinfo automatically, so I don't have to type these commands now.

nano ~/.bash_profile

export PATH=~/bin/sratoolkit.3.0.0-mac64/bin:$PATH

source ~/.bash_profile

conda activate bioinfo

doctor.py

------------------------------------------------------------------------------------------------------------------------------------------------------

I am getting this error after running doctor.py. I am installing on a Mac OS with an M1 chip.

![editederror](https://user-images.githubusercontent.com/75187110/174170073-d69ce4d2-d8d8-47cb-afde-e10b5b349a57.jpg)

I tried to follow the directions outlined[ here](https://github.com/biostars/biostar-handbook/discussions/209) but this did not work either. Strangely, running the below command caused doctor.py to no longer run at all and any new shell I open gives me a warning message about zsh being the new apple default, although my shell is still in bash.

_echo 'export PATH=~/bin/sratoolkit.3.0.0-mac64/bin:$PATH' > ~/.bashrc_

Running _which fastq-dump_ outputs: /Users/me/miniconda3/envs/bioinfo/bin/fastq-dump

However, running _fastq-dump_ before running _conda activate bioinfo_ outputs:/Users/me/bin/sratoolkit.3.0.0-mac64/bin/fastq-dump. 

I am assuming this means that it is using the old version instead of the one I tried to download. 

Running _~/bin/sratoolkit.3.0.0-mac64/bin/fastq-dump_ works

**Comment by louis-mak0000** [comment, marked as answer]
I ran into the same issue as well--after fastq-dump failed, I downloaded the latest ubuntu version of sratoolkit and added the new directory to the path (outlined [here](https://github.com/biostars/biostar-handbook/discussions/209)), and then doctor.py stopped working. The issue is probably the line of code below, which overwrote the original .bashrc file and replace with only the 'export PATH=...':

echo 'export PATH=~/bin/sratoolkit.3.0.0-mac64/bin:$PATH' > ~/.bashrc

I did the following and the installation passed doctor.py. and nothing strange so far:

1. Restored the original .bashrc with this (discussed [here](https://askubuntu.com/questions/404424/how-do-i-restore-bashrc-to-its-default)):
/bin/cp /etc/skel/.bashrc ~/

2. Followed the steps on the biostar handbooks related to modifying .bashrc: 
Step 3:
~/miniconda3/condabin/conda init bash
curl http://data.biostarhandbook.com/install/bashrc.txt >> ~/.bashrc

3. Added the sratoolkit directory to the path (the ">>" appends, instead of overwrites)
echo 'export PATH=~/bin/sratoolkit.3.0.0-ubuntu64/bin:$PATH' >> ~/.bashrc

4. Restarted the terminal and run doctor.py

Hope it helps (I'm also a newbie).

**Comment by goddy93** [comment]
@louis-mak0000 Can you please check the links, "Restored the original .bashrc with this (discussed **link** ) and other link "added the new directory to the path (**link)**. 

These links seems to be broken.

**Reply by louis-mak0000** [reply]
Thanks for pointing those out! I've updated the links. Should work now.

**Reply by goddy93** [reply]
thanks. it worked
